packages/kampus-ui/src/InternalLink.tsx

```diff
@@ -0,0 +1,10 @@
+import { Link as RouterLink } from "@remix-run/react";
```

Comment:
kampus-ui shouldn't depend on `remix-react`, let's move this to Pano.

Reply:
My first solution was the same as your solution but I got confused because of the Link component then I change my component code. Okay, I will update my code with this solution. 

[packages/kampus-ui/src/link.tsx](../blob/main/packages/kampus-ui/src/Link.tsx)

Reply:
Ok, i am ok with living it here as we already have another component that depends on remix, we will fix this by going over all the links we have in kampus-ui, and unifying them in the future.